compiler/test/dotc/pos-test-recheck.exludes

```diff
@@ -0,0 +1,7 @@
+# Cannot compensate dealiasing due to false result dependency
```

Comment:
Why is the file name extension `.exludes` instead of `.excludes`? TestSources.scala repeats this typo.